packages/paste-website/src/pages/content/product-style-guide.mdx

```diff
@@ -192,56 +198,106 @@ Coming in the next release
 - *Behavior, dialed, color, artifact, center, license* (for both verb and noun), *-ize* endings, *program* (for all meanings)
 
 #### Bold and italics
-Coming in the next release
+
+##### Bold 
+**Do** use bold type sparingly, to draw attention to the most important word, number, or short phrase in your copy. 
+- Bold type can help a user scan the page more effectively by making landmarks more prominent.
+
+**Do** consider using bold type to refer to UI elements in instructional content. E.g.
+- *Select the location from **Region/State**.*
+- *Select **Continue** to review your order.*
+
+**Don't** use bold multiple times in body copy or for a whole sentence/paragraph. It dilutes its effect.
+- Only **bold the most important information the user needs to retain** and deploy it only if user testing flags a need. 
+- Consider if putting an important sentence on a new line will do a better job of making it stand out than making it bold. 
+
+##### Italics
+**Avoid** using italic type wherever possible. 
+- If you need to draw attention to a word, use bold type.
+- If you need to quote a phrase, use quotation marks "like this." 
```

Comment:
Are there any cases where italicized type is appropriate? I know we use them in the docs sometimes when we make "_e.g._" examples, might be useful to include kind of a whitelist example of where they are useful or appropriate?

Reply:
I'm keen to stick with *avoid italics in product content* - I've dropped a clarifying sentence in there to explain that while folks might see it used elsewhere, when it comes to UI and products, we're keeping things simple - bold for emphasis, italics for quoted text.